src/Resources/ModelRestResource.php

```diff
@@ -533,8 +530,15 @@ private function fetchItems($from, $to, RhubarbDateTime $since = null, $asSummar
             $this->filterModelCollectionForModifiedSince($collection, $since);
         }
 
-        $pageSize = ($to - $from) + 1;
-        $collection->setRange($from, $pageSize);
+        if ($rangeStart > 0 || $rangeEnd !== false) {
+            if ($rangeEnd === false) {
+                $count = sizeof($collection);
+                $pageSize = $count - $rangeStart;
```

Comment:
when i read the description of the pr i thought "i wonder if he'll include something to give me the last x of a resource" ^_^